jobs/cronjob-ldap-group-sync.yml

```diff
@@ -62,65 +84,72 @@ objects:
                 command:
                   - "/bin/bash"
                   - "-c"
-                  - "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm || :"
+                  - oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm
+                    $([ -s /etc/config/whitelist.txt ] && echo --whitelist=/etc/config/whitelist.txt)
```

Comment:
I presume you tested and found this to be working, but a quick check with `oc process` of this template shows that `&` gets expanded to `\u0026` ... again, this may not be a problem, but wanted to ensure this is expected behavior

```
> oc process -f jobs/cronjob-ldap-group-sync.yml ...  | grep 'oc adm groups'
 "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm $([ -s /etc/config/whitelist.txt ] \u0026\u0026 echo --whitelist=/etc/config/whitelist.txt)"
```

Reply:
Yes, I tested and all is well. I'm not sure why it gets expanded like this, but in the resulting cronjob it is back to "&"